src/audio/volume/volume.c

```diff
@@ -235,21 +233,24 @@ static void volume_ramp(struct comp_dev *dev)
 	 */
 	cd->vol_ramp_active = true;
 
-	/* inc/dec each volume if it's not at target for active channels */
+	/* Remaining ramp time in milliseconds */
```

Comment:
Sorry, a mistake in comment, should be "current ramp time". This is from my first approach that didn't work. I'll fix..

Reply:
Done.